doc(release) Prepare release notes for cloud release 24.04 March (#3176)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md

title: Centreon Cloud - notes de release
---

## March 29, 2024

> ### Custom check command definition

Reviewer: ```suggestion
### Custom check command definition

```

   You now have the ability to define your own check commands, allowing for customized monitoring configurations.
To maintain system integrity, this feature is secured by a whitelist mechanism (this requires a poller update), ensuring only authorized commands are executed.

> ### Dashboards Beta:

Reviewer: ```suggestion
### Dashboards Beta

```

   As usual, lots of improvements to the dashboard feature with this release:
- Navigation in dashboards has been enhanced with a new search feature, a listing presentation and a better responsiveness.
- Existing widgets have been improved, based on feedback from beta testers. It is now more intuitive to select metrics or to select the widget you need.
- A new widget named “Resource Table” is available, allowing you to visualize the status of a set of selected resources.

Important note: this is the last time Centreon Dashboards is delivered as a beta version. In the next version, Centreon Dashboards will be generally available.

> ### Miscellaneous improvements

Reviewer: ```suggestion
### Miscellaneous improvements

```

> ### Custom Check Command definition
>    You now have the ability to define your own check commands, allowing for personalized monitoring configurations.
> To maintain system integrity, this feature is secured by a whitelist mechanism (requires a poller update), ensuring only authorized commands are executed.
> 
> ### Dashboards Beta:
>    As usual, lots of improvements of the Dashboard feature with this release:
> - Navigation in dashboards has been enhanced with a new search feature, a listing presentation and a better responsiveness.
> - Existing widgets have been improved, based on beta tester feedbacks. It is now more intuitive to select metrics or to select the widget you need.
> - A new widget named “Resource Table” is available allowing to visualize status of a set of selected resources.
> 
> Important note: this is the last time Centreon Dashboards is delivered as a beta version. On next version, Centreon Dashboards will be generally available.
> 
> ### Miscenaleous improvements
> - You can now display any page of Centreon, and in particular dashboards, in Full Screen Mode.
> - It is now possible to use semicolons in command macros (and in particular in passwords).

Reviewer: ```suggestion
### Custom check command definition
   You now have the ability to define your own check commands, allowing for customized monitoring configurations.
To maintain system integrity, this feature is secured by a whitelist mechanism (this requires a poller update), ensuring only authorized commands are executed.

### Dashboards Beta:
   As usual, lots of improvements to the dashboard feature with this release:
- Navigation in dashboards has been enhanced with a new search feature, a listing presentation and a better responsiveness.
- Existing widgets have been improved, based on feedback from beta testers. It is now more intuitive to select metrics or to select the widget you need.
- A new widget named “Resource Table” is available, allowing you to visualize the status of a set of selected resources.

Important note: this is the last time Centreon Dashboards is delivered as a beta version. In the next version, Centreon Dashboards will be generally available.

### Miscellaneous improvements
- You can now display any page in Centreon, and in particular dashboards, in full screen mode.
- It is now possible to use semicolons in command macros (and in particular in passwords).
```

title: Centreon Cloud - notes de release
---

## March 29, 2024

### Custom check command definition

>    You now have the ability to define your own check commands, allowing for customized monitoring configurations.

Reviewer: ```suggestion
You now have the ability to define your own check commands, allowing for customized monitoring configurations.
```

To maintain system integrity, this feature is secured by a whitelist mechanism (this requires a poller update), ensuring only authorized commands are executed.

### Dashboards Beta

>    As usual, lots of improvements to the dashboard feature with this release:

Reviewer: ```suggestion
As usual, lots of improvements to the dashboard feature with this release:
```

- Navigation in dashboards has been enhanced with a new search feature, a listing presentation and a better responsiveness.
- Existing widgets have been improved, based on feedback from beta testers. It is now more intuitive to select metrics or to select the widget you need.
- A new widget named “Resource Table” is available, allowing you to visualize the status of a set of selected resources.

> Important note: this is the last time Centreon Dashboards is delivered as a beta version. In the next version, Centreon Dashboards will be generally available.

Reviewer: ```suggestion
**Important note**: this is the last time Centreon Dashboards is delivered as a beta version. In the next version, Centreon Dashboards will be generally available.
```

title: Centreon Cloud - notes de release
---

## March 29, 2024

### Custom check command definition

You now have the ability to define your own check commands, allowing for customized monitoring configurations.

> To maintain system integrity, this feature is secured by a whitelist mechanism (this requires a poller update), ensuring only authorized commands are executed.

Reviewer: ```suggestion
To maintain system integrity, this feature is secured by a whitelist mechanism, ensuring only authorized commands are executed.
As an important note, whether or not you plan to use this feature, it may introduce a security vulnerability if you do not update your poller by following [this procedure](../installation/poller-update-upgrade.md).
```
